accounts/views.py

```diff
@@ -429,8 +430,14 @@ def describe(request):
                 return render(request, 'accounts/confirm_delete_undescribed_files.html', tvars)
             elif "delete_confirm" in request.POST:
                 for f in form.cleaned_data["files"]:
-                    os.remove(files[f].full_path)
-                    remove_uploaded_file_from_mirror_locations(files[f].full_path)
+                    try:
+                        os.remove(files[f].full_path)
+                        remove_uploaded_file_from_mirror_locations(files[f].full_path)
+                    except OSError as e:
+                        if e.errno == errno.ENOENT:
+                            logger.error("Failed removing file " + str(e) + " (no such file or directory)")
```

Comment:
This is really picky, but...
- with loggers we should use query parameters - `logger.error("foo %s", param)`
- `str(e)` is already going to include "no such file or directory" so we probably don't need the additional comment in brackets
- Grammar - "Failed to remove file"

Reply:
Fixed https://github.com/MTG/freesound/pull/948/commits/bbd0f63e19780cb8ec64b65fdec1f1ea8846ec2e :)